samples/websocket_connect.md

```diff
@@ -4,6 +4,12 @@
 
 This sample makes an MQTT connection via Websockets and then disconnects. On startup, the device connects to the server via Websockets and then disconnects right after. This sample is for reference on connecting via Websockets. This sample demonstrates the most straightforward way to connect via Websockets by querying the AWS credentials for the connection from the device's environment variables or local files.
 
+If you want to use custom auth (or static creds, or basic auth, etc) instead,
```

Comment:
We should explicitly write that these are websocket with custom auth (and others).